wrappers/swift/Askar/Package.swift

```diff
@@ -0,0 +1,29 @@
+// swift-tools-version: 5.7
+import PackageDescription
+
+let package = Package(
+    name: "Askar",
+    platforms: [
+        .iOS(.v15)
+    ],
+    products: [
+        .library(
+            name: "Askar",
+            targets: ["Askar"]),
+    ],
+    dependencies: [
+    ],
+    targets: [
+        .target(
+            name: "Askar",
+            dependencies: ["AskarFramework"]),
+        .testTarget(
+            name: "AskarTests",
+            dependencies: ["Askar"]),
+        .binaryTarget(
+            name: "AskarFramework",
+            url: "https://github.com/hyperledger/aries-framework-swift/releases/download/binary-release-askar-v0.2.7/AskarFramework.xcframework.zip",
```

Comment:
Is this url definitive? There is already an xcframework on the releases of this library. Maybe in the future it can just use that one (right now it also includes Android builds though).

Reply:
This link should be changed later. I'm thinking of adding a workflow job for this.